charts/castai-cluster-controller/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: castai-cluster-controller
 description: CAST AI cluster controller deployment chart.
 type: application
-version: 0.13.0
+version: 0.14.0
 appVersion: "v0.10.1"
```

Comment:
as you have changed cluster-controller, I would suggest to update appVersion in one go when it is released

Reply:
Got it, will changed app version here once it is released